Dockerfile

```diff
@@ -6,10 +6,12 @@ RUN corepack enable
 WORKDIR /app
 COPY package.json pnpm-lock.yaml ./
 COPY .husky/install.mjs ./.husky/
+COPY prisma ./prisma/
 
 FROM base AS dev-deps
 WORKDIR /app
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
+RUN npx prisma generate
```

Comment:
use pnpm?

Reply:
Since there is no separate pnpm command for `prisma generate`, `npx prisma generate` is used here, as there is no need to import .prisma files every time. Maybe consider adding a new command that only has `prisma generate`.

Reply:
`pnpm prisma generate` works. When we use pnpm, we can directly run commands provided by installed packages without needing to explicitly invoke a separate command

Reply:
Ok, done